### PR TITLE
fix(docs): update framework button disabled state

### PR DESCRIPTION
**Pull request by calebpollman**
#### Description of changes
Update framework chooser button disabled state

**Screenshots**

**Light Mode:**
_Before:_
Screen Shot 2022-06-11 at 3 22 34 PM
_After:_
Screen Shot 2022-06-11 at 3 22 53 PM

**Dark Mode:**
_Before:_
Screen Shot 2022-06-11 at 3 22 19 PM
_After:_
Screen Shot 2022-06-11 at 3 23 12 PM

#### Issue #, if available
NA

#### Description of how you validated changes
Manual tests

#### Checklist

- [x] PR description included

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/styles/docs/sidebar.scss [hunk 1]
```diff
@@ -52,11 +51,15 @@
...
-  
+
   &.current {
     background-color: var(--amplify-components-button-hover-background-color);
     border-color: var(--amplify-components-button-active-border-color);
   }
+
+  &.disabled {
+    opacity: 0.6;
+  }
```
Comment (calebpollman) [inline docs/src/styles/docs/sidebar.scss:62]: This is the only relevant change in this file, rest is prettier

File: docs/src/styles/docs/sidebar.scss [hunk 2]
```diff
@@ -52,11 +51,15 @@
...
   gap: var(--amplify-space-xs);
-  
+
   &.current {
     background-color: var(--amplify-components-button-hover-background-color);
     border-color: var(--amplify-components-button-active-border-color);
   }
+
+  &.disabled {
+    opacity: 0.6;
```
Comment (zchenwei) [inline docs/src/styles/docs/sidebar.scss:61]:
```suggestion
    opacity: var(--amplify-opacities-60);
```

**Review by zchenwei — APPROVED** [review]
LGTM! Only a small suggestion.

**Review by joebuono — APPROVED** [review]

**Comment by dbanksdesign** [comment]
Let's hold off on this PR. We should fix the disabled state of the button for Amplify UI rather than doing it on the docs site only. I made a PR (#2084) to fix the disabled button color along with some updates to the default dark mode.

**Comment by calebpollman** [comment]
> Let's hold off on this PR. We should fix the disabled state of the button for Amplify UI rather than doing it on the docs site only. I made a PR (#2084) to fix the disabled button color along with some updates to the default dark mode.

Nice. Didn't realized the issue was with the `Button` component itself 👍

**Comment by calebpollman** [comment]
Closing in favor of https://github.com/aws-amplify/amplify-ui/pull/2084

**Pull request closed without merging**